src/studio/src/designer/frontend/ux-editor/components/config/editModal/EditHeaderSize.tsx

```diff
@@ -0,0 +1,47 @@
+import Select from 'react-select';
+import React from 'react';
+import { PropertyLabel, selectStyles } from '../../../utils/render';
+import type { IGenericEditComponent } from '../componentConfig';
+
+enum HeaderSize {
+  S = 'h4',
+  M = 'h3',
+  L = 'h2',
+}
+
+export const EditHeaderSize = ({ handleComponentChange, component, language }: IGenericEditComponent) => {
+  const sizes = [
+    { value: HeaderSize.S, label: language['ux_editor.modal_header_type_h4'] },
+    { value: HeaderSize.M, label: language['ux_editor.modal_header_type_h3'] },
+    { value: HeaderSize.L, label: language['ux_editor.modal_header_type_h2'] },
```

Comment:
Ville gjort dette via en funksjon. altså ikke accesser `language` direkte, men ha en wrapper funksjon. Enklere å refaktorere når vi går over til ett bedre system for språkhåndtering (messageformat).